client/index.jsx

```diff
@@ -1,7 +1,8 @@
 // eslint-disable-next-line import/extensions
-import React from 'react';
-import ReactDOM from 'react-dom';
```

Comment:
wtf ES lint doesn't let you use ES6 syntax?

Reply:
I was stuck on this for like 2 hours... I kept getting errors from using import-from syntax